helpers/contactHelper.js

```diff
@@ -0,0 +1,78 @@
+const getSortedHelpOtions = (res) => {
+	const helpOptions = [
+		{
+			optionValue: 'Aufgaben',
```

Comment:
I wasn't sure if I should translate the option value to english because it was in german before.